### PR TITLE
Fix warning when memory_limit = -1

### PR DESCRIPTION
**Pull request by ghost**
### Description:

When memory_limit = -1 (no limit) there is a warning, I did not find  issue 

### Review

* [ ] Functional review done
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php [hunk 1]
```diff
@@ -41,7 +41,10 @@ public function execute()
         $memoryLimit = SettingsServer::getMemoryLimitValue();
         $comment = $memoryLimit . 'M';
 
-        if ($memoryLimit >= $this->minimumMemoryLimit) {
+        if(false === $memoryLimit) {
+            $status = DiagnosticResult::STATUS_OK;
+            $comment = 'No memory limit';
```
Comment (Findus23) [inline plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php]: Hi,

Could you maybe expand a bit on that comment and make it translatable (like the other ones)?

Thanks for you contribution.

**Review by Findus23 — CHANGES_REQUESTED** [review]

File: plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php (hunk 1 above)
Comment (ghost) [reply plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php]: When memory_limit = -1 there is a warning ( screenshot ) 

Here we can see $memoryLimit is set to false when no limit

https://github.com/matomo-org/matomo/blob/4.x-dev/core/SettingsServer.php#L201

I am a newbie on matomo i check the translation guide line

Capture d’écran de 2021-02-12 10-49-14

Comment (Findus23) [reply plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php]: Sorry, I wasn't precise enough. I just meant make the comment a full sentence, just so noone is confused if this is a suggestion or a statement.

File: plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php [hunk 2]
```diff
@@ -41,7 +41,10 @@ public function execute()
         $memoryLimit = SettingsServer::getMemoryLimitValue();
         $comment = $memoryLimit . 'M';
 
-        if ($memoryLimit >= $this->minimumMemoryLimit) {
+        if(false === $memoryLimit) {
+            $status = DiagnosticResult::STATUS_OK;
+            $comment = $this->translator->translate('Installation_SystemCheckNoMemoryLimitSet');
```
Comment (sgiehl) [inline plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php]: You also need to add the default text for this to `plugins\Installation\lang\en.json`

Comment (ghost) [reply plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php]: Ok done.

**Review by sgiehl — APPROVED** [review]

**Comment by sgiehl** [comment]
@okom3pom  Thanks for your contribution!

**Comment by ghost** [comment]
Thank you too for the job !

**Pull request merged**